tabcmd/commands/project/create_project_command.py

```diff
@@ -26,11 +26,12 @@ def run_command(args):
         else:
             project_path = None
 
+        logger.info("===== Creating project '{}' on the server...".format(args.name))
         top_level_project = TSC.ProjectItem(args.name, args.description, None,
                                             project_path)
         try:
             project_item = server.projects.create(top_level_project)
-            logger.info('Successfully created a new project called: {}'.format(top_level_project.name))
+            logger.info('===== Succeeded')
```

Comment:
Are we having info message as only **_succeeded_** to match whats shown in Tabcmd1?

Reply:
yep - we can add more info in a debug message too, but I want the info messages to match